policies/test_test.rego

```diff
@@ -13,21 +13,31 @@ test_needs_tests_with_results {
 }
 
 test_needs_tests_with_results_mixed {
-	deny == {{"msg": "Found tests without results"}} with data.test as [{}, {"result": "SUCCESS"}]
+	deny == {{"msg": "Found tests without results"}} with data.test as [{}, {"test1": {"result": "SUCCESS"}}]
 }
 
 test_success_data {
-	count(deny) == 0 with data.test as [{"result": "SUCCESS"}]
+	count(deny) == 0 with data.test.test1 as {"result": "SUCCESS"}
 }
 
 test_failure_data {
-	deny == {{"msg": "All tests did not end with SUCCESS"}} with data.test as [{"result": "FAILURE"}]
+	deny == {{"msg": "All tests did not end with SUCCESS"}} with data.test.test1 as {"result": "FAILURE"}
+		with data.config.policy as {"non_blocking_checks": []}
 }
 
 test_error_data {
-	deny == {{"msg": "All tests did not end with SUCCESS"}} with data.test as [{"result": "ERROR"}]
+	deny == {{"msg": "All tests did not end with SUCCESS"}} with data.test.test1 as {"result": "ERROR"}
+		with data.config.policy as {"non_blocking_checks": []}
 }
 
 test_mix_data {
-	deny == {{"msg": "All tests did not end with SUCCESS"}} with data.test as [{"result": "SUCCESS"}, {"result": "FAILURE"}, {"result": "ERROR"}]
+	deny == {{"msg": "All tests did not end with SUCCESS"}} with data.test.successfull as {"result": "SUCCESS"}
```

Comment:
Definitely nitpicking, but can we make all `with` statements always line up? As it is, the first one appears to have a different importance than the other ones.

Reply:
Split into separate steps and added comments.

Reply:
I did that, `opa fmt` would disagreed

Reply:
:grimacing: